lib/StoryblokComponent.astro

```diff
@@ -11,21 +14,30 @@ interface Props {
 const { blok, ...props } = Astro.props;
 
 /**
- * convert blok components to camel case to match vite-plugin-storyblok
+ * convert blok components to camel case to match vite-plugin-storyblok-components
  */
-const key = camelcase(blok.component);
+let key = camelcase(blok.component);
 
-if (!(key in components)) {
-  /**
-   * Component not found! Log an error with details on the blok name that could not be found
-   * TODO: Add support for a placeholder component as a fallback?
-   */
-  throw new Error(
-    `Component could not be found for blok "${blok.component}"! Is it defined in astro.config.mjs?`
-  );
-}
+const componentFound: boolean = key in components;
+
+let Component: AstroComponentFactory;
 
-const Component = components[key];
+if (!componentFound) {
+  if (!options.enableFallbackComponent)
+    throw new Error(
+      `Component could not be found for blok "${blok.component}"! Is it defined in astro.config.mjs?`
+    );
+  else {
+    if (options.customFallbackComponent) {
```

Comment:
I wonder if this logic could be moved up into the Vite plugin?

If the local `FallbackComponent.astro` component is resolved in the plugin it could be used as a default value for the `customFallback` export in `virtual:storyblok-components`